### ISSUE TITLE
Feature Request: Option to Download missing Lora's referenced in Civitai user Generated Info

### ISSUE THREAD

**Issue by chuckkay** [sunnyark/civitai-shortcut#71]
Description: When browsing the CivitAi Shortcut User gallery section for a good checkpoint, I often come across great-looking image, in the user gallery section. Upon clicking on them and viewing the Image Info "Generated Info" section, I find that it is filled with creative prompts that I would love to just send to txt2img. However, these prompts often contain Lora's that I either do not have or have trouble finding. It would be great if the CivitAi Shortcut extension could parse the prompt for any missing Lora's and provide the option to download them.

What do you think of this feature request? 😊

**Comment by sunnyark** [comment]
Exactly, it's a great idea. However, the problem is that CivitAi doesn't offer such a feature. It's difficult to search through all models, and even the filenames of the used Lora aren't certain. It's a bit tricky situation.
